File: src/main/java/io/github/support/terminal/platform/app/domains/core/api/telegram/models/Update.java
package io.github.support.terminal.platform.app.domains.core.api.telegram.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.ToString;
import lombok.experimental.Accessors;

@Data
@ToString
@Accessors(chain = true)
public class Update {

    @JsonProperty("update_id")
    private Integer updateId;


    @JsonProperty("message")
    private Message message;
}

File: src/main/kotlin/io/github/support/terminal/application/domains/integrations/telegram/models/Update.kt
package io.github.support.terminal.application.domains.integrations.telegram.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class Update(
        @JsonProperty("update_id") val updateId: Int,
        @JsonProperty("message") val message: Message?
)
